settings.gradle.kts

```diff
@@ -1,3 +1,4 @@
 rootProject.name = "web5"
 include("common", "crypto", "dids", "credentials")
```

Comment:
```suggestion
include("common", "crypto", "dids", "credentials", "sdjwt")
```

Reply:
Actually, separated everything into single lines and sorted alphabetically.